script.js

```diff
@@ -61,9 +81,13 @@
 // Write your solution below.
 // =======================================================================
 
-
-
-
+// var randomInt = Math.floor(Math.random() * 11);
```

Comment:
randomInt can be 0 here...

Reply:
Thanks Chee Kean. Sorry forgot to exclude 0.